src/app/workflow/dag/cwl-viewer/cwl-viewer.service.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { CwlViewerService } from './cwl-viewer.service';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { Dockstore } from '../../../shared/dockstore.model';
+
+describe('Service: commonwl', () => {
```

Comment:
Leaving the name as the generated default for consistency.

Reply:
Apparently the new CLI generates a different naming nowadays.  Should be "Service: CWLViewer" or similar

Reply:
Not sure about everyone else, but I use IDE (which uses Angular CLI) to generate all my components/services .ts, .spec.ts, .html, .scss.  For the most part, we've been trying to keep the naming somewhat consistent.

Reply:
I've noticed it in my IDE but haven't been in the habit of using it; I'll try to start.